### PR TITLE
Bug fix postgres tests

### PR DESCRIPTION
**Pull request by jmbrunskill**
# 👩🏻‍💻 What does this PR do?

Fixes postgres tests with the immunisation program work

## 💌 Any notes for the reviewer?

Went with a single context for now, for all immunisations.
We could add something in the future around parent programs in context but for now keeping it simple.

# 🧪 Testing

`cargo test --features=postgres`

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [X] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: server/repository/src/migrations/v2_01_00/activity_log.rs [hunk 1]
```diff
@@ -7,6 +7,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
             r#"
                 ALTER TYPE activity_log_type ADD VALUE 'ASSET_PROPERTY_CREATED';
                 ALTER TYPE activity_log_type ADD VALUE 'VACCINE_COURSE_CREATED';
+                ALTER TYPE activity_log_type ADD VALUE 'PROGRAM_CREATED';
```
Comment (clemens-msupply) [inline server/repository/src/migrations/v2_01_00/activity_log.rs:10]: Are they used anywhere? can't find any references to the string values in the code, or how does it work?

**Review by clemens-msupply — APPROVED** [review]
Looks good, just one question...

File: server/repository/src/migrations/v2_01_00/activity_log.rs (hunk 1 above)
Comment (jmbrunskill) [reply server/repository/src/migrations/v2_01_00/activity_log.rs:10]: It's generated by the enum with SCREAMING_SNAKE_CASE...
Here:https://github.com/msupply-foundation/open-msupply/blob/34feb1426e4484af539f72c1adcabb73b58c151e/server/repository/src/db_diesel/activity_log_row.rs#L29
Would be nice if we could use enums without Postgres Needing this. I feel like we should be able to do a mapping to TEXT Field in diesel...

Comment (clemens-msupply) [reply server/repository/src/migrations/v2_01_00/activity_log.rs:10]: Ah yes right, should have known...

**Pull request merged**